src/lib.rs

```diff
@@ -47,64 +47,64 @@ pub fn example() {
     let mut y = 10;
 
     let label = Label::new()
-        .position(x, y)
-        .size(400, 16)
-        .text("Test Label")
-        .place(&mut window);
```

Comment:
I'll merge it in, but I need you to fix the formatting in this file back to the original. In my opinion, the indenting that rustfmt did here was unnecessary, because it indented the code further away, which takes away valuable line space for longer lines of code. I also want you to squash the first 2 commits you did into one, because the history of removing the .bk files is not necessary.